_episodes/07-tasks-and-automation.md

```diff
@@ -1,19 +1,114 @@
+
+Please note that you will see the name of the task and can type something different in this window. However, these changes only for the name won't take effect. To change the name of a task, you need to go to Task Actions and select Rename task.
+
+You can add or update a description in this task list window. The box in the middle will list the fuctions (or tasks) for this particular task. If there are none, this window will be empty.
+
+Actions include:
+- Add: Add a new function or task
+- Duplicate: Clone a function or task already in the list
+- Delete: Delete a function or task
+- Edit: change an existing function or task
+
+You can add a new task by clicking on the plus button. You can add any type of function available in the MarcEditor Tools and Edit Shortcuts menu options. This includes the RDA Helper, build a new field, change the case of a MARC field, or generate cutters. You can even include a list of functions (tasks) from another Task.
+
+Click on save at the left bottom corner of the Task List window to save your Task. From there you can close the Task Manager window. To run a task, go to Tools, then Assigned Taks, Current Tasks.
+
+If you have a lot of Tasks, it does help to assign them to groups to more easily find what you need. If you don't use Groups, the Task Manager will list all your Tasks in alphabetical order and it might be difficult to see them on especially on small screens such as laptops.
+
+>## Create and run a new Task that runs the RDA helper and removes 9XX fields
+>
+>1. .
+>
+> > ## Solution
+> > 1. Go to Tools in the upper menu in the MarcEditor
+> > 2. Select Manage Tasks
+> > 3. In the Task Manager window, in Task Actions, select New Task
+> > 4. Click Select.
+> > 5. Name your Task
+> > 6. In the Task List window, you can describe this task.
+> > 7. Click on the plus button and add RDA Helper.
+> > 8. Click on the plus button and add a new Add Delete Task Field
+> > 9. In the new window to delete a field, add 9XX and click on Delete field.
+> > 10. Save Task
+> > 11. Close the Task Manager
+> > 12. Go to Tools, then Assigned Tasks, and select your task that you just created
+> > 13. Check your MARC data. Are the 9xx gone? Does every record now have the appropriate RDA fields?
+> {: .solution}
+{: .challenge}
+
+## Managing your Task
+To change an existing task, go to Tools and Manage Task. This will open the Task Manager window. From here you can rename tasks, export, import them, or assign a Task to a Group.
+
+>## Add a proxy to the 856 40$u in your Task
+>
+>1. After you run your Task, your MARC records should have your institution's proxy before the url. Remember to check your data to see if there are any proxy stems present.You will need to remove those before running your Task. If the MARC data has different types of electronic urls (resource, related resource, etc.) as noted in the MARC field 856 indicators, you will need to account for that in your Task.
```

Comment:
```suggestion
>1. After you run your Task, your MARC records should have your institution's proxy before the URL. Remember to check your data to see if there are any proxy stems present. You will need to remove those before running your Task. If the MARC data has different types of electronic URLs (resource, related resource, etc.) as noted in the MARC field 856 indicators, you will need to account for that in your Task.
```

Reply:
Does the data require this check? If so we would want to walk them through it. Otherwise we might add this detail as a note for consideration, but not an actual task to do in the lesson?